release.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+function help() {
+    echo "Usage: sh release.sh <RELEASE_VERSION>"
+    echo ""
+    echo "Parameters:"
+    echo "  RELEASE_VERSION: SemVer Version number for the release"
+}
+
+semver_pattern="^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)$"
+
+RELEASE_VERSION=${1}
+
+[[ -z "${RELEASE_VERSION}" ]] && echo "ERROR: RELEASE_VERSION is empty" && help  && exit 1
+[[ ! "${RELEASE_VERSION}" =~ $semver_pattern ]] && echo "ERROR: RELEASE_VERSION does not match the SemVer specification" && help && exit 1
+
+printf "\n%s\n" "Create new branch release/${RELEASE_VERSION}"
+git checkout -b release/${RELEASE_VERSION}
```

Comment:
The release should be performed based on the develop branch. This command would just create a branch from whatever branch you are currently on. Therefore you should ensure to use the develop branch as base branch for the release.
```suggestion
git checkout -b release/${RELEASE_VERSION} origin/develop
```